src/components/Layout/example/MenuItem.tsx

```diff
@@ -0,0 +1,227 @@
+        Петрик Пяточкин
+      </Typography>
+      <Typography variant="body1">petrik@gmail.com</Typography>
+    </Link>
+  </Button>
+)
+
+const FavoriteButton = ({ closeVisible }: { closeVisible: () => void }) => (
+  <Button
+    sx={{
+      width: 'fit-content',
+      height: '40px',
+      borderRadius: '20px',
+      fontSize: '14px',
+      padding: '7px 10px'
+    }}
+  >
+    <Link to="favorite" className={styles2.linkAccount} onClick={closeVisible}>
+      <FavoriteBorderIcon
+        sx={{
+          fontSize: '26px',
+          position: 'absolute',
+          top: '0px',
+          left: '-12px'
+        }}
+      />
+      <Typography variant="body1" sx={{ fontSize: '18px', fontWeight: '600' }}>
+        Улюблене
+      </Typography>
+    </Link>
+  </Button>
+)
+
+export const MenuItem = ({
+  toggleOpen,
+  isActive,
+  setIsActive,
+  isLessThan600px
+}: any) => {
+  const isLoggedIn = useSelector((state: RootState) => state.auth.isLoggedIn)
+
+  return (
+    <Fragment>
+      {isLessThan600px && (
+        <motion.li
+          className={styles.exampleLi}
+          variants={variants}
+          whileHover={{ scale: 1.1 }}
+          whileTap={{ scale: 0.95 }}
+        >
+          <SearchGlobal isActive={isActive} setIsActive={setIsActive} />
+        </motion.li>
+      )}
+
+      {isLoggedIn && (
+        <Fragment>
+          <motion.li
+            className={styles.exampleLi}
+            variants={variants}
+            whileHover={{ scale: 1.1 }}
+            whileTap={{ scale: 0.95 }}
+          >
+            <AccountButton closeVisible={() => console.log('Closing menu')} />
+          </motion.li>
+          <motion.li
+            className={styles.exampleLi}
+            variants={variants}
+            whileHover={{ scale: 1.1 }}
+            whileTap={{ scale: 0.95 }}
+          >
+            <FavoriteButton closeVisible={() => console.log('Closing menu')} />
+          </motion.li>
+        </Fragment>
+      )}
+
+      <motion.li
+        className={`${styles.exampleLi} ${styles.listNavLine}`}
+        variants={variants}
+        whileHover={{ scale: 1.1 }}
+        whileTap={{ scale: 0.95 }}
+      >
+        <CatalogLink />
+      </motion.li>
+
+      <motion.li
+        className={`${styles.exampleLi} ${styles.listNavLine}`}
+        variants={variants}
+        whileHover={{ scale: 1.1 }}
+        whileTap={{ scale: 0.95 }}
+      >
+        <ReviewLink />
+      </motion.li>
+
+      <motion.li
+        className={`${styles.exampleLi} ${styles.listNavLine}`}
+        variants={variants}
+        whileHover={{ scale: 1.1 }}
+        whileTap={{ scale: 0.95 }}
+      >
+        <AboutUsLink toggleOpen={toggleOpen} />
+      </motion.li>
+
+      <motion.li
+        className={`${styles.exampleLi} ${styles.listNavLine}`}
+        variants={variants}
+        whileHover={{ scale: 1.1 }}
+        whileTap={{ scale: 0.95 }}
+      >
+        <CooperationLink />
+      </motion.li>
```

Comment:
move this `li` to component like `MenuItemLink` and reuse it there, don't repeat code

Reply:
Done